code/game/gamemodes/cult/hell_universe.dm

```diff
@@ -50,6 +50,19 @@ In short:
 
 	suspend_alert = 1
 
+	OnEnter1()
+	OnEnter2()
+	OnEnter3()
+	OnEnter4()
+	OnEnter5()
```

Comment:
Profiling is done by proc. Having 5 different key functions in one proc absolutely wrecks profiling tools.